modules/upgrade-scanner-roxctl-371.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// upgrade-scanner.adoc
```

Comment:
Is this the full path to the assembly?

Reply:
Oops, no - didn't realize I was supposed to include the full path